api/lib/infrastructure/repositories/assessment-repository.js

```diff
@@ -32,8 +30,7 @@ module.exports = {
         qb.where({ userId });
         qb.where('state', '=', 'completed');
         qb.andWhere(function() {
-          this.where({ type: null })
```

Comment:
ça veut dire qu'il n'y a plus d'assessment avec pas de type ?

Reply:
Ça vient de changer, thx to @DaRiviere et @jonathanperret :)